Updated and Corrected Job.md (#34445)
kubernetes/website: content/en/docs/concepts/workloads/controllers/job.md

<!-- overview -->

A Job creates one or more Pods and will continue to retry execution of the Pods until a specified number of them successfully terminate.

> As pods are successfully completed, the Job tracks the successful completions.  When a specified number

Reviewer: ```suggestion
As pods are successfully completed, the Job tracks the successful completions.  When a specified number
```
The original adheres to the [Kubernetes style guide on simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

of successful completions is reached, the task (i.e. Job) is complete.  Deleting a Job will clean up
the Pods it created. Suspending a Job will delete its active Pods until the Job
is resumed again.

A simple case is to create one Job object in order to reliably run one Pod to completion.
The Job object will start a new Pod if the first Pod fails or is deleted (for example
due to a node hardware failure or a node reboot).

> You can also use a Job to run multiple Pods in parallel. See [this](https://kubernetes.io/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).

Reviewer 2: We prefer to use `/docs/tasks/...` link instead of `https://kubernetes.io/docs/...`.
```suggestion
You can also use a Job to run multiple Pods in parallel. See [this](/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).
```

Reviewer: @utkarsh-singh1 , we use reference paths for links.
@Sea-n 's comment is [correct](https://github.com/kubernetes/website/pull/34445/files#r903501103)

> You can also use a Job to run multiple Pods in parallel. See [this](https://kubernetes.io/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).

Reviewer: ```suggestion
You can also use a Job to run multiple Pods in [parallel](/docs/concepts/workloads/controllers/job/#parallel-jobs). 
```
There are multiple types of parallel jobs, the one suggested is for a work queue.
I suggest to add the links to the examples in the section that describes parallels jobs: https://kubernetes.io/docs/concepts/workloads/controllers/job/#parallel-jobs
You can add a line with the example for the work queue parallel job

## Running an example Job

> Here is an example Job config.  It computes value of π upto 2000 places and prints it out.

Reviewer: ```suggestion
Here is an example Job config.  It computes π to 2000 places and prints it out.
```
I prefer the original as it adheres to the [Kubernetes style guide for simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
